Makefile

```diff
@@ -28,7 +28,7 @@ nodepool_create:
 resource_apply:
 	$(PROMBENCH_CMD) gke resource apply -a ${AUTH_FILE} \
 		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} \
-		-v PR_NUMBER:${PR_NUMBER} -v RELEASE:${RELEASE} \
+		-v PR_NUMBER:${PR_NUMBER} -v RELEASE:${RELEASE} -v DOMAIN_NAME:prombench.prometheus.io \
```

Comment:
should be an env var. Should try to avoid hard codding string as much as possible. 
It might see that this will never change, but my experience is that at some point it will :)